BridgeCareApp/BridgeCareCodeFirst/Services/SimulationRepository.cs

```diff
@@ -0,0 +1,65 @@
+﻿using BridgeCare.Models;
+using System;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Web;
+using BridgeCare.EntityClasses;
+using BridgeCare.Interfaces;
+using BridgeCare.ApplicationLog;
+
+namespace BridgeCare.Services
+{
+    public class SimulationRepository : ISimulation
```

Comment:
Avoid the Repository name unless we are following the repository design pattern.

SimulationProvider